### PR TITLE
rename system contracts:

### PR DESCRIPTION
**Pull request by olexiybuyanskyy**
eosio.bios => rem.bios
eosio.system => rem.system
eosio.wrap => rem.wrap
eosio.token => rem.token
eosio.msig => rem.msig

rename system accounts:

eosio.bpay => rem.bpay
eosio.msig => rem.msig
eosio.names => rem.names
eosio.ram => rem.ram
eosio.ramfee => rem.ramfee
eosio.saving => rem.saving
eosio.stake => rem.stake
eosio.token => rem.token
eosio.vpay => rem.vpay
eosio.rex => rem.rex
eosio.null => rem.null
eosio.prods => rem.prods

rename system account from
eosio => rem

rename utilities:

cleos => remcli
nodeos => remnode
keosd => remvault

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: contracts/contracts/rem.system/src/producer_pay.cpp [hunk 1]
```diff
@@ -277,11 +277,11 @@ namespace eosiosystem {
       require_auth( payer );
       const auto to_per_stake_pay = amount.amount * 0.7; //TODO: move to constants section
       const auto to_per_vote_pay  = share_pervote_reward_between_producers(amount.amount * 0.2); //TODO: move to constants section
-      const auto to_remme         = amount.amount - (to_per_stake_pay + to_per_vote_pay);
+      const auto to_rem         = amount.amount - (to_per_stake_pay + to_per_vote_pay);
```
Comment (mark-tsyrulnyk) [inline contracts/contracts/rem.system/src/producer_pay.cpp]: @olexiybuyanskyy do we need these changes?

File: CMakeModules/package.cmake [hunk 2]
```diff
@@ -1,8 +1,8 @@
-set(VENDOR "block.one")
-set(PROJECT_NAME "eosio")
-set(DESC "Software for the EOS.IO network")
-set(URL "https://github.com/eosio/eos")
-set(EMAIL "support@block.one")
+set(VENDOR "remme.io")
+set(PROJECT_NAME "remprotocol")
+set(DESC "Software for the REMME.IO network")
+set(URL "https://github.com/Remmeauth/remchain")
```
Comment (mark-tsyrulnyk) [inline CMakeModules/package.cmake]: @olexiybuyanskyy https://github.com/Remmeauth/remprotocol

**Review by mark-tsyrulnyk — APPROVED** [review]

File: contracts/contracts/rem.system/src/producer_pay.cpp (hunk 1 above)
Comment (me-vadim) [inline contracts/contracts/rem.system/src/producer_pay.cpp]: broken formatting, add two spaces

File: contracts/contracts/rem.system/src/producer_pay.cpp [hunk 3]
```diff
@@ -277,11 +277,11 @@ namespace eosiosystem {
...
       if( amount.amount > 0 ) {
         token::transfer_action transfer_act{ token_account, { {_self, active_permission} } };
-        if( to_remme > 0 ) {
-           transfer_act.send( _self, saving_account, asset(to_remme, core_symbol()), "Remme Savings" );
+        if( to_rem > 0 ) {
+           transfer_act.send( _self, saving_account, asset(to_rem, core_symbol()), "Rem Savings" );
```
Comment (me-vadim) [inline contracts/contracts/rem.system/src/producer_pay.cpp]: Do we need to change memo? 'Remme' is an organization not account

**Review by me-vadim — APPROVED** [review]

**Pull request merged**